torch/nn/modules/fold.py

```diff
@@ -42,7 +43,10 @@ class Fold(Module):
     Shape:
         - Input: :math:`(N, C, L_{in})`
         - Output: :math:`(N * C * \prod(kernel\_size), L_{out},)` where
-          :math:`L_{out} = floor((L_{in} + 2 * padding - dilation * (kernel\_size - 1) - 1) / stride + 1)`
+
+    .. math::
+        L_{out} = floor((L_{in} + 2 * padding - dilation *
```

Comment:
Can you add `\text` on `padding`, `dilation`, `kernel_size` and `stride` here too?